### PR TITLE
Tox

### PR DESCRIPTION
**Pull request by cemsbr**
Close #317

### CODE REVIEW

**Comment by cemsbr** [comment]
@beraldoleal, enable Scrutinizer in this repo, please.

File: .scrutinizer.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+checks:
+    python:
+        code_rating: true
+        duplicate_code: true
+filter:
```
Comment (diraol) [inline .scrutinizer.yml]: @cemsbr this files/directories are from kytos, not kytos-utils.

**Comment by beraldoleal** [comment]
@cemsbr done.

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist = py36
+
+[testenv]
+whitelist_externals = rm
+commands=
+    ; Force packaging even if setup.{py,cfg} haven't changed
+    rm -rf ./*.egg-info/
+    coverage run setup.py test
+    pylama tests setup.py kytos
```
Comment (diraol) [inline tox.ini]: kytos?

File: .scrutinizer.yml (hunk 1 above)
Comment (cemsbr) [inline .scrutinizer.yml]: If you mean `['kytos/*', 'tests/*']`, they are the same from kytos.

File: tox.ini (hunk 2 above)
Comment (cemsbr) [inline tox.ini]: Yes, the source code is in the folder _kytos/_.

File: .scrutinizer.yml (hunk 1 above)
Comment (diraol) [inline .scrutinizer.yml]: Sorry! I have made a mistake about the repo, I was thinking about kytos-napps. This is ok!

File: tox.ini (hunk 2 above)
Comment (diraol) [inline tox.ini]: Sorry! I have made a mistake about the repo, I was thinking about kytos-napps. This is ok!

**Pull request merged**